sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -220,9 +220,7 @@ Azure::Security::KeyVault::Keys::DeleteKeyOperation KeyClient::ResumeDeleteKey(
     Azure::Core::Context const& context) const
 {
   Azure::Security::KeyVault::Keys::DeleteKeyOperation operation(m_pipeline, resumeToken);
-  // Need to fix this to use context directly. See:
-  // https://github.com/Azure/azure-sdk-for-cpp/issues/2091
-  operation.Poll(context.GetApplicationContext());
```

Comment:
Have we fixed all LROs that had this issue (some may not have had the comment)? @vhvb1989 can you also verify?

Reply:
yes - to my best understanding and effort